allure-results/c4fac028-6fa9-4c4f-b34e-ff65063ed5ff-testsuite.xml

```diff
@@ -0,0 +1,91 @@
+<?xml version='1.0'?>
```

Comment:
Can we remove the allure-results folder, and add to ignore?

Reply:
I have added to the .gitignore along with the other test results.

Reply:
aw okay, you wanted me to add the folder as well??? It was taking the folders and the files too which I could see being greyed out.